Automattic/woocommerce-payments
# Add red notification badge next to payments menu item

Fixes #1435 

#### Changes proposed in this Pull Request

Adds a notification badge next to the payments menu until a user views `ConnectAccountPage` or successfully connects to Stripe account.

No tests were added since I think this feature is a non-critical feature, plus a test would require mocking for `WC_Payments_Admin` which looks extensive. Let me know if you think of otherwise, I would glad to provide tests if required.

#### Screenshot

<img src="https://cdn-std.droplr.net/files/acc_1119691/odyiFt" width="300">

#### Testing instructions

Note: You can install the plugin [woocommerce-admin-test-helper](https://github.com/woocommerce/woocommerce-admin-test-helper/archive/refs/tags/v0.1.0.zip) to be able to quickly delete options, or access the database directly if that's preferable.

1. Checkout this branch in your local repository. 
2. Delete the following options if they exist:
    - `wcpay_menu_badge_hidden`
    - `jetpack_options` (This disconnects WPCOM connection)
2. Navigate to any screen in wp-admin apart from the Payments menu.
3. Verify that a red notification badge with the count `1` appears next to the payment menu.
4. Click on the payments menu.
5. Click on `Set up`.
6. Click back (or optionally finish setup, doesn't matter).
5. Verify that the badge disappears.

-------------------
